package.json

```diff
@@ -32,7 +32,7 @@
     "ngb:lint": "ng lint ng-bootstrap && yarn schematics:lint",
     "ngb:test": "ng test ng-bootstrap --code-coverage --source-map true --progress false --watch false",
     "ngb:tdd": "ng test ng-bootstrap --source-map false",
-    "ngb:e2e": "concurrently --success first --kill-others \"yarn e2e-app:serve --prod --no-watch\" \"yarn ngb:e2e-playwright BROWSER=chromium && yarn ngb:e2e-playwright BROWSER=firefox && yarn ngb:e2e-playwright BROWSER=webkit\"",
+    "ngb:e2e": "concurrently --success first --kill-others \"yarn e2e-app:serve --prod --no-watch -c playwright\" \"yarn ngb:e2e-playwright BROWSER=chromium && yarn ngb:e2e-playwright BROWSER=firefox && yarn ngb:e2e-playwright BROWSER=webkit\"",
```

Comment:
I think you can remove `--prod` and `--no-watch` and add 
`"browserTarget": "e2e-app:build:production"` in the `angular.json` configuration